app/models/user.rb

```diff
@@ -23,6 +23,8 @@ class User < ApplicationRecord
   validates_format_of :phone_number, with: /\A[0-9]{10}\z/, allow_nil: true,
                                      message: 'must be a valid US phone number'
 
+  validates :name, presence: { message: 'must provide a name' }
```

Comment:
I'm gonna comment this out for now, since it will invalidate lots of old models in production.

Reply:
Hmmm, I thought that validation only ran when you actually tried to update or create a modelled thing, and was intending this to be backwards compatible. What specifically are you afraid of breaking? We should probably test it anyways

Reply:
When you edit notification preferences, that modifies the user.  Users without names aren't able to change their preferences.  I tested it.